### PR TITLE
Classic switch

### PR DESCRIPTION
**Pull request by MichaelDJansen**

### CODE REVIEW

**Review by timbielawski — APPROVED** [review]

File: admin/pages/settings.php [hunk 1]
```diff
@@ -80,6 +81,9 @@
                 <label>PWA App Endpoint</label>
                 <input type="text" name="appEndpoint" value="<?= $theme->getAppEndpoint() ?>" />
                 <div class="spacer-20"></div>
+
+                <input type="checkbox" style="margin: 0 10px 0 0px;" name="showClassicSwitch" <?= $theme->getShowClassicSwitch() ? 'checked' : '' ?> /> Show Classic Site Switch
```
Comment (dustanfranks) [inline admin/pages/settings.php]: 2012 Called, they want their in-line styles back.
```suggestion
                <input type="checkbox" name="showClassicSwitch" <?= $theme->getShowClassicSwitch() ? 'checked' : '' ?> /> Show Classic Site Switch
```

File: admin/pages/themes.php [hunk 2]
```diff
@@ -50,7 +50,7 @@
 						<h2 class="title">Available Mobile App Themes</h2>
 						<div class="spacer-30"></div>
 
-						<div class="themes" style="width: 220px;">
```
Comment (dustanfranks) [inline admin/pages/themes.php:53]: Is this code for a development branch? 
```suggestion
						<div class="themes">
```

File: export/class-export-settings.php [hunk 3]
```diff
@@ -35,7 +35,7 @@ protected function get_application_manager()
 				require_once(WMP_PLUGIN_PATH.'frontend/class-application.php');
 			}
 
-            return new WMobilePack_Application();
+            return new WMobilePack_Application(plugin_dir_url(__FILE__));
```
Comment (dustanfranks) [inline export/class-export-settings.php]: This should be cast to a variable that's assigned at the root level folder of the plugin

File: inc/class-theme.php [hunk 4]
```diff
@@ -1476,4 +1476,25 @@ public function setAppEndpoint($appEndpoint)
...
+    /**
+     * Set the value of showClassicSwitch
+     *
+     * @return  self
+     */ 
+    public function setShowClassicSwitch($showClassicSwitch)
+    {
+        $this->showClassicSwitch = $showClassicSwitch;
+
+        return $this;
```
Comment (dustanfranks) [inline inc/class-theme.php:1497]: Why do you need to return $this in this setter method? What is the context in which it is being called?

File: wordpress-mobile-pack.php [hunk 5]
```diff
@@ -20,7 +20,7 @@
 function wmobilepack_frontend_init() {
 
     require_once('frontend/class-application.php');
-    new WMobilePack_Application();
+    new WMobilePack_Application(plugin_dir_url(__FILE__));
```
Comment (dustanfranks) [inline wordpress-mobile-pack.php:25]: You are reusing this function as an argument repeatedly, as per earlier comment, you should cache this to a variable declared in the plugin manifest file.

**Review by dustanfranks — APPROVED** [review]
Nothing untoward, just a few line items that could be improved.

**Review by vjustinpietersen — APPROVED** [review]

**Pull request merged**